server/src/dbschema.js

```diff
@@ -94,6 +105,17 @@ exports.createTables = function () {
         if (count != 1) {
           throw new Error("Should have deleted one row");
         }
+        return db.getConnection().then(([conn, done]) => {
```

Comment:
Well inserting a row without a head and a body also works because head and body allow nulls at the moment, and we do need the tables to be created first before running the patcher. It does seem to make sense to run the sanity test to see if the tables were created properly before running the patcher.